src/fromager/commands/bootstrap.py

```diff
@@ -196,6 +197,8 @@ def bootstrap(
                     f"Could not produce a pip compatible constraints file. Please review {constraints_filename} for more details"
                 )
 
+    logger.info("match_py_req LRU cache: %r", resolver.match_py_req.cache_info())
```

Comment:
I guess this is something to understand how much LRU cache we use in the bootstrap? Do we want this all bootstrap logs?

Reply:
+1 for debug log. I am also fine with merging the PR as it and create a follow up PR to move it to debug.